source/upgrade.txt

```diff
@@ -13,9 +13,12 @@ Upgrade Driver Versions
 Overview
 --------
 
-On this page, you can learn about any changes you might need to make to
-your application to upgrade your driver to a new version without loss of
-functionality.
+On this page, you can learn how to upgrade your driver to a new version. This page also 
+includes the changes you must make to your application to upgrade your driver 
+without a loss of functionality, if applicable.
```

Comment:
Suggestion:

I think "a loss of functionality" doesn't identify what loses the functionality as well as the following phrase:

```suggestion
without losing functionality, if applicable.
```

Implied in this statement is "without your application losing functionality" whereas the current version doesn't imply that relationship.